### PR TITLE
cmd/kube-controller-manager: add myself as owner

### PR DESCRIPTION
**Pull request by sttts**
Having done quite a lot in the command plumbing in the last months here, the testserver, the KEP around component config, adding myself as approver (only to the command, not the controllers!):

```
$ git log --pretty=format:"%h%x09%an%x09%ad%x09%s" cmd/kube-controller-manager/ | grep Schimanski
8aa0eefce8	Dr. Stefan Schimanski	Thu Aug 30 19:20:19 2018 +0200	kube-controller-manager: disable authn/z on insecure port
e209b643a7	Dr. Stefan Schimanski	Thu Jul 5 15:45:58 2018 +0200	kube-controller-manager: add test server and test serving
4cc3b2e6bb	Dr. Stefan Schimanski	Tue Aug 7 11:14:45 2018 +0200	kube-controller-manager: enable secure loopback
5fa8b4b2ae	Dr. Stefan Schimanski	Tue Aug 7 11:11:23 2018 +0200	kube-controller-manager: enable delegated authz/authn if secure port is enabled
eb27b61cdb	Dr. Stefan Schimanski	Tue Aug 7 11:13:01 2018 +0200	kube-controller-manager: enable secure ports 10257, deprecate insecure port
c2724793e8	Dr. Stefan Schimanski	Tue Aug 7 07:34:13 2018 +0200	Update bazel
1d9a896066	Dr. Stefan Schimanski	Thu Aug 16 20:47:15 2018 +0200	apiserver: move controller-manager's insecure config into apiserver
5843bfc6b1	Dr. Stefan Schimanski	Thu Jul 5 15:47:28 2018 +0200	kube-controller-manager: create self-signed certs
8686feea6d	Dr. Stefan Schimanski	Thu Jul 5 15:45:26 2018 +0200	kube-controller-manager: add stopCh plumbing
d79cf25497	Dr. Stefan Schimanski	Fri Jun 29 21:17:38 2018 +0200	Update external k8s.io/metrics imports
5483ab7679	Dr. Stefan Schimanski	Thu Feb 8 19:37:08 2018 +0100	Update generated files
f4564ea0b8	Dr. Stefan Schimanski	Thu Feb 8 19:28:31 2018 +0100	controller-manager: add SecureServingOptions
bec617f3cc	Dr. Stefan Schimanski	Thu Nov 9 09:36:32 2017 +0100	Update generated files
012b085ac8	Dr. Stefan Schimanski	Wed Nov 8 23:34:54 2017 +0100	pkg/apis/core: mechanical import fixes in dependencies
cad0364e73	Dr. Stefan Schimanski	Mon Oct 16 14:04:55 2017 +0200	Update bazel
7773a30f67	Dr. Stefan Schimanski	Mon Oct 16 13:41:50 2017 +0200	pkg/api/legacyscheme: fixup imports
2b8e938128	Dr. Stefan Schimanski	Mon Jan 23 10:02:54 2017 +0100	Update generated files
82826ec273	Dr. Stefan Schimanski	Mon Jan 23 15:46:05 2017 +0100	pkg/util/flag: move to k8s.io/apiserver
a6b2ebb50c	Dr. Stefan Schimanski	Fri Jan 20 14:05:41 2017 +0100	pkg/flag: make feature gate extensible and split between generic and kube
56d60cfae6	Dr. Stefan Schimanski	Thu Jan 19 13:39:13 2017 +0100	pkg/util: move flags from pkg/util/config to pkg/util/flags
3d9449a353	Dr. Stefan Schimanski	Tue Jan 17 11:38:25 2017 +0100	genericapiserver: fix imports
2dff13f332	Dr. Stefan Schimanski	Mon Dec 5 10:56:55 2016 +0100	Update generated files
24e24fc7bb	Dr. Stefan Schimanski	Thu Nov 17 14:21:12 2016 +0100	Add verb support to gc and namespace controllers
458d2b2fe4	Dr. Stefan Schimanski	Thu Nov 17 14:19:03 2016 +0100	Add verb support for discovery client
bc7523a775	Dr. Stefan Schimanski	Tue Oct 20 20:04:09 2015 +0100	Make controller manager node monitor constant public
```

### CODE REVIEW

**Comment by mikedanese** [comment]
lgtm
/approve

File: cmd/kube-controller-manager/OWNERS [hunk 1]
```diff
@@ -63,5 +64,6 @@ reviewers:
 - thockin
 - timothysc
 - wojtek-t
+- sttts
```
[the remark this answers is not in the document]
Comment (sttts) [reply cmd/kube-controller-manager/OWNERS]: Oh right, one of the 50 people on that list :-)

[the remark this answers is not in the document]
Comment (sttts) [reply cmd/kube-controller-manager/OWNERS]: fixed

**Pull request merged**